specification/cognitiveservices/data-plane/MetricsAdvisor/preview/v1.0/MetricsAdvisor.json

```diff
@@ -0,0 +1,6312 @@
+          "description": "created time",
+          "type": "string",
+          "readOnly": true
+        },
+        "modifiedTime": {
+          "format": "date-time",
+          "description": "modified time",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "AlertResultList": {
+      "required": [
+        "@nextLink",
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "@nextLink": {
+          "type": "string"
+        },
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AlertResult"
+          }
+        }
+      }
+    },
+    "AnomalyProperty": {
+      "required": [
+        "anomalySeverity"
+      ],
+      "type": "object",
+      "properties": {
+        "anomalySeverity": {
+          "description": "anomaly severity",
+          "enum": [
+            "Low",
+            "Medium",
+            "High"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "Severity",
+            "modelAsString": true
+          }
+        },
+        "anomalyStatus": {
+          "description": "anomaly status\r\n\r\nonly return for alerting anomaly result",
+          "enum": [
+            "Active",
+            "Resolved"
+          ],
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "AnomalyResult": {
+      "required": [
+        "dimension",
+        "property",
+        "timestamp"
+      ],
+      "type": "object",
+      "properties": {
+        "metricId": {
+          "format": "uuid",
+          "description": "metric unique id\r\n\r\nonly return for alerting anomaly result",
+          "type": "string",
+          "readOnly": true
+        },
+        "anomalyDetectionConfigurationId": {
+          "format": "uuid",
+          "description": "anomaly detection configuration unique id\r\n\r\nonly return for alerting anomaly result",
+          "type": "string",
+          "readOnly": true
+        },
+        "timestamp": {
+          "format": "date-time",
+          "description": "anomaly time",
+          "type": "string"
+        },
+        "createdTime": {
+          "format": "date-time",
+          "description": "created time\r\n\r\nonly return for alerting result",
+          "type": "string",
+          "readOnly": true
+        },
+        "modifiedTime": {
+          "format": "date-time",
+          "description": "modified time\r\n\r\nonly return for alerting result",
+          "type": "string",
+          "readOnly": true
+        },
+        "dimension": {
+          "description": "dimension specified for series",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "property": {
+          "$ref": "#/definitions/AnomalyProperty"
+        }
+      }
+    },
+    "AnomalyResultList": {
+      "required": [
+        "@nextLink",
```

Comment:
`@nextLink` should be optional (and not sent if there is no next page)

Reply:
fixed.